projects/project-1/assignment/assignment_1_short.md

```diff
@@ -0,0 +1,3235 @@
+
+|        | Funct. | iFunct. | Trans. | Symm. | Asymm. | Ref. | Irref. |
+|--------|--------|---------|--------|-------|--------|------|--------|
+| **Funct.**   | -      | OK      | X<sup>NS</sup>    | OK    | OK     | OK   | OK     |
+| **iFunct.**  | OK     | -       | X<sup>NS</sup>    | OK    | OK     | OK   | OK     |
+| **Trans.**   | X<sup>NS</sup>    | X<sup>NS</sup>     | -      | OK    | X<sup>NS</sup>    | OK   | X<sup>NS</sup>    |
+| **Symm.**    | OK     | OK      | OK     | -     | X<sup>UNSAT</sup> | OK   | OK     |
+| **Asymm.**   | OK     | OK      | X<sup>NS</sup>    | X<sup>UNSAT</sup>| -      | X<sup>UNSAT</sup>| OK    |
+| **Ref.**     | OK     | OK      | OK     | OK    | X<sup>UNSAT</sup> | -    | X<sup>UNSAT</sup> |
+| **Irref.**   | OK     | OK      | X<sup>NS</sup>    | OK    | OK     | X<sup>UNSAT</sup>| -      |
+
+\
+Trans.-Funct (NS)
+
+Suppose that a relation R is both transitive and functional. This creates a complex role, which known proofs show creates undecidability.
+
+Trans.-iFunct (NS)
+
+Suppose that a relation R is both transitive and inverse functional. This creates a complex role, which known proofs show creates undecidability.
+
+Asymm-Trans (NS)
+
+Suppose that a relation R is both transitive and inverse functional. This creates a complex role, which known proofs show creates undecidability.
+
+Asymm-Symm (UNSAT) (done by JB)
+
+Ref-Asymm (UNSAT)
+
+Suppose that a relation R is both reflexive and asymmetric. By reflexivity, for any x, x is related to itself by R, i.e. xRx. By asymmetry, for any x and any y, if it is the case that xRy, then it is not the case that yRx. But for reflexivity, the only entities in R are x. That is, y=x. Plugging that in, we both have that xRx for reflexivity, and that it is not the case that xRx for asymmetry. This is thus not satisfiable.
```

Comment:
It might be helpful to emphasize how reflexive and asymmetric properties inherently contradict each other as a further clarification.